host/xtest/regression_4000.c

```diff
@@ -41,6 +45,12 @@ static void xtest_tee_test_4008(ADBG_Case_t *Case_p);
 static void xtest_tee_test_4009(ADBG_Case_t *Case_p);
 static void xtest_tee_test_4010(ADBG_Case_t *Case_p);
 static void xtest_tee_test_4011(ADBG_Case_t *Case_p);
+#ifdef CFG_SECURE_KEY_SERVICES
+static void xtest_tee_test_4003_pkcs11(ADBG_Case_t *Case_p);
```

Comment:
The 4003_no_xts and 4003_xts exists for historical reason, they could be merged into a single 4003 but none has bothered to do that yet.

Please use a separate number instead of 4003_pkcs11.

Reply:
https://github.com/OP-TEE/optee_test/pull/258

Reply:
i'll rebase.

This test reuses the 4003 data. How to best share the test vectors, if moving the pcsk11 test to a **regression_4100.c** which sound reasonable also to me?

*alt1/* i plan to move the test vectors in **regression_4100_data.h** (where xts vectors are) and have it included in both **regression_4000.c** and **regression_4100.c**  (make each source having a static copy of the vectors). If think it is not very nice.

*alt2/*  Export all test vector in the xtest (remove the `static` attrib). I think it is not nice.

Reply:
It seems there doesn't exist any nice and easy solution to this so we need to be a bit pragmatic about this. Since you're indeed using `ciph_cases` to drive `xtest_tee_test_4003_pkcs11()` I'm OK with keeping this particular function in this file even if the test number would be in the 41xx series. We can have a comment in regression_4100.c that one of the sequence number are used in regression_4000.c for practical reasons.

Reply:
sounds good. ok, thanks for the proposal.